pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -386,6 +387,19 @@ def _do_catch(self, pokemon, encounter_id, catch_rate_by_ball, is_vip=False):
                         'exp': sum(response_dict['responses']['CATCH_POKEMON']['capture_award']['xp'])
                     }
                 )
+
+                # We could refresh here too, but adding 3 saves a inventory request
+                candy = inventory.candies().get(pokemon.num)
```

Comment:
@TheSavior no internally it will remap Pidgeotto to Pidgey and maintain one count. This is also what is received in the response to a GET_INVENTORY request. The logic is to get the first evolution for the passed in pokemon number and then retrieve the number of candies for that. That logic is in the inventory.py file.